src-gui/ui/src/app/modules/switches/port-graph/port-graph.component.ts

```diff
@@ -8,6 +8,7 @@ import { LoaderService } from 'src/app/common/services/loader.service';
 import { IslDataService } from 'src/app/common/services/isl-data.service';
 import { CommonService } from 'src/app/common/services/common.service';
 import { MessageObj } from 'src/app/common/constants/constants';
+import {ActivatedRoute} from '@angular/router';
```

Comment:
spaces near the curly braces would be nice as it is in other places. 
The same in other places

Reply:
thanks, fixed it